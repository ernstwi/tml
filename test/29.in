init foo
foo -> bar
    sync { }
