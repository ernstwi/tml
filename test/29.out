Parse error at FILE "test/29.in" 3:12: Unexpected '}'. Expected valid identifier
